Rakefile

```diff
@@ -113,10 +113,10 @@ task 'gem:native' do
   if RUBY_PLATFORM =~ /darwin/
     FileUtils.touch 'grpc_c.32.ruby'
     FileUtils.touch 'grpc_c.64.ruby'
-    system "rake cross native gem RUBY_CC_VERSION=2.4.0:2.3.0:2.2.2:2.1.5:2.0.0 V=#{verbose} GRPC_CONFIG=#{grpc_config}"
+    system "rake cross native gem RUBY_CC_VERSION=2.5.0:2.4.0:2.3.0:2.2.2:2.1.6:2.0.0 V=#{verbose} GRPC_CONFIG=#{grpc_config}"
```

Comment:
Did you change our set of Darwin environments as well ? As in, I'm not sure you want to move from 2.1.5 to 2.1.6 on Darwin, do you ?

Reply:
Darwin environments were changed in earlier PR: https://github.com/grpc/grpc/pull/13930/files. I didn't see adjusting the Darwin 2.1.x to use 2.1.6 as strictly necessary, I just did that only for consistency (as I understand 2.1.6 is ABI-compatible with 2.1.5).

Reply:
FTR manual smoke testing seems fine (grpc built against ruby-2.1.6 still loads fine on ruby 2.1.0). I'm fairly confident that building against 2.1.6 is non-breaking too since, e.g., up through now I believe that any ABI incompatibilities between ruby <i>teeny</i> version releases would break the grpc package (https://github.com/grpc/grpc/blame/master/src/ruby/lib/grpc/grpc.rb#L16)